go/stats/counters.go

```diff
@@ -224,3 +224,13 @@ func NewMultiCountersFunc(name string, labels []string, f CountersFunc) *MultiCo
 	}
 	return t
 }
+
+var escaper = strings.NewReplacer(".", "\\.", "\\", "\\\\")
```

Comment:
I think this is going to be very expensive. I believe these functions get called on every query. Can you benchmark the before and after?